packages/nuka/README.md

```diff
@@ -0,0 +1,38 @@
+[![Nuka Carousel — Formidable, We build the modern web](https://raw.githubusercontent.com/FormidableLabs/nuka-carousel/main/Nuka-Hero.png)](https://formidable.com/open-source/)
```

Comment:
For groqd, I pointed this link at the actual groqd docs. Might do the same here but point to nuka's.

Reply:
Good idea!